test/test_wave.py

```diff
@@ -78,89 +79,149 @@ def mesh_factory(n):
-                    c * sym.div(sym_v) + sym_f,
-                    make_obj_array([c]) * sym.grad(dim, sym_u))
+        nodes = discr.nodes().with_queue(queue)
 
         def sym_eval(expr, t):
             return sym.EvaluationMapper({"x": nodes, "t": t})(expr)
 
-        u = sym_eval(sym_u, 0.)
-        v = sym_eval(sym_v, 0.)
-        f = sym_eval(sym_f, 0.)
+        t_check = 1.23456789
+
+        u = sym_eval(sym_u, t_check)
+        v = sym_eval(sym_v, t_check)
 
-        from mirgecom.wave import wave_operator
-        rhs = wave_operator(discr, c=c, w=flat_obj_array(u, v))
-        rhs[0] = rhs[0] + f
+        fields = flat_obj_array(u, v)
 
-        expected_rhs = sym_eval(sym_rhs, 0.)
+        rhs = wave_operator(discr, c=c, w=fields)
+        rhs[0] = rhs[0] + sym_eval(sym_f, t_check)
 
-        err = np.max(np.array([la.norm((rhs[i] - expected_rhs[i]).get(), np.inf)
-                for i in range(dim+1)]))
-        eoc_rec.add_data_point(1./n, err)
+        expected_rhs = sym_eval(sym_rhs, t_check)
+
+        eoc_rec.add_data_point(1./n, max_inf_norm(rhs - expected_rhs))
 
         if visualize:
             from grudge.shortcuts import make_visualizer
             vis = make_visualizer(discr, discr.order)
-            vis.write_vtk_file("result_{n}.vtu".format(n=n),
-                    [
-                        ("u", u),
-                        ("v", v),
-                        ("rhs_u_actual", rhs[0]),
-                        ("rhs_v_actual", rhs[1:]),
-                        ("rhs_u_expected", expected_rhs[0]),
-                        ("rhs_v_expected", expected_rhs[1:]),
+            vis.write_vtk_file("wave_accuracy_{order}_{n}.vtu".format(order=order,
+                        n=n), [
+                            ("u", fields[0]),
+                            ("v", fields[1:]),
+                            ("rhs_u_actual", rhs[0]),
+                            ("rhs_v_actual", rhs[1:]),
+                            ("rhs_u_expected", expected_rhs[0]),
+                            ("rhs_v_expected", expected_rhs[1:]),
                         ])
 
     print("Approximation error:")
     print(eoc_rec)
     assert(eoc_rec.order_estimate() >= order - 0.5 or eoc_rec.max_error() < 1e-11)
+
+    # Check stability
+
+    mesh = mesh_factory(8)
+
+    from grudge.eager import EagerDGDiscretization
+    discr = EagerDGDiscretization(cl_ctx, mesh, order=order)
+
+    nodes = discr.nodes().with_queue(queue)
+
+    def sym_eval(expr, t):
+        return sym.EvaluationMapper({"x": nodes, "t": t})(expr)
+
+    def get_rhs(t, w):
+        result = wave_operator(discr, c=c, w=w)
+        result[0] += sym_eval(sym_f, t)
+        return result
+
+    t = 0.
+
+    u = sym_eval(sym_u, t)
+    v = sym_eval(sym_v, t)
+
+    fields = flat_obj_array(u, v)
+
+    from mirgecom.integrators import rk4_step
+    dt = timestep_scale/order**2
+    for istep in range(10):
+        fields = rk4_step(fields, t, dt, get_rhs)
+        t += dt
```

Comment:
Did this fail before the fix? (Just making sure.)

Reply:
Yep! See [here](https://github.com/illinois-ceesd/mirgecom/runs/851228389), e.g.:

```
Traceback (most recent call last):
187
  File "/home/runner/work/mirgecom/mirgecom/test/test_wave.py", line 227, in test_wave
188
    assert(err < max_err)
189
AssertionError: assert 116748.05780039357 < 1.8771041427775335
```